src/Sulu/Bundle/SecurityBundle/UserManager/UserManager.php

```diff
@@ -685,7 +685,9 @@ private function processEmail(UserInterface $user, $email, $contact = null)
                 $user->setEmail($email);
             }
         } else {
-            if ($email && 0 !== \strcasecmp($email, $user->getEmail()) && !$this->isEmailUnique($email)) {
+            $emailChanged = $email && (null === $user->getEmail() || 0 !== \strcasecmp($email, $user->getEmail()));
+
+            if ($emailChanged && !$this->isEmailUnique($email)) {
```

Comment:
Then would go this way:

```suggestion
            $currentEmail = $user->getEmail() ?? '';
            $hasEmailChanged = $email && 0 !== \strcasecmp($email, $currentEmail);

            if ($hasEmailChanged && !$this->isEmailUnique($email)) {
```